src/main/java/duke/logic/commands/results/Calenderable.java

```diff
@@ -2,6 +2,9 @@
 
 import duke.model.lists.EventList;
 
+/**
+ * Interface representing a command result containing a calender.
```

Comment:
Interface name does not suggest that is exclusive to command result, so command result should be omitted from the comments.